### ISSUE TITLE
Replace one-element array flex-array member in fs/reiserfs/reiserfs.h

### ISSUE THREAD

**Issue by GustavoARSilva** [KSPP/linux#290]
```c
diff --git a/fs/reiserfs/reiserfs.h b/fs/reiserfs/reiserfs.h
index 1bccf6a2e908..65924fcdd28b 100644
--- a/fs/reiserfs/reiserfs.h
+++ b/fs/reiserfs/reiserfs.h
@@ -2374,7 +2374,7 @@ struct virtual_node {
 struct direntry_uarea {
        int flags;
        __u16 entry_count;
-       __u16 entry_sizes[1];
+       __u16 entry_sizes[];
 } __attribute__ ((__packed__));
```

**Comment by GustavoARSilva** [comment]
Patch: https://lore.kernel.org/linux-hardening/ZJN9Kqhcs0ZGET%2F8@work/

**Comment by kees** [comment]
Fixed in commit 76d2ceda723c9.
